src/main/java/net/pms/dlna/DLNAMediaAudio.java

```diff
@@ -38,11 +39,17 @@
 	private static final Logger LOGGER = LoggerFactory.getLogger(DLNAMediaAudio.class);
 	private AudioProperties audioProperties = new AudioProperties();
 
+	public static final int BITRATE_DEFAULT = 8000;
+	public static final int NUMBEROFCHANNELS_DEFAULT = 2;
+	public static final int BITSPERSAMPLE_DEFAULT = 16;
+	public static final int AUDIODELAY_DEFAULT = 0;
+	public static final int SAMPLEFREQUENCY_DEFAULT = 48000;
```

Comment:
@Sami32 I really don't have an opinion about what the default should be except that it should be what is most likely to be correct. My idea is always to try hard to do actual parsing and not having to rely on defaults. My "impression" is that 48000 is more common, but that's not based on any real facts but is just made up of which files I've happened to see most lately. If you'd asked me 10 years ago I would definitely have said 44100 though.